app/src/ui/changes/changed-file.tsx

```diff
@@ -9,6 +9,8 @@ import { Checkbox, CheckboxValue } from '../lib/checkbox'
 
 const GitIgnoreFileName = '.gitignore'
 
+const RestrictedFileExtensions = ['.cmd', '.exe', '.bat', '.sh']
```

Comment:
I don't fully understand the problem, but should we be at all worried that this is playing whack-a-mole?